### PR TITLE
Misc plural fixes

### PR DESCRIPTION
**Pull request by Qrox**
#### Summary
SUMMARY: None

#### Purpose of change
Fix a few plurals in json files.

#### Testing
Ran the unit test and no error was reported.

### CODE REVIEW

File: data/json/items/ammo/8x40mm.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   {
     "id": "8mm_caseless",
     "type": "AMMO",
-    "name": { "str": "8x40mm caseless" },
+    "name": { "str": "8x40mm caseless", "str_pl": "8x40mm caselesses" },
```
Comment (ZhilkinSerg) [inline data/json/items/ammo/8x40mm.json:15]: I am not sure this one is correct.

Comment (Qrox) [reply data/json/items/ammo/8x40mm.json:15]: What's your suggestion?

Comment (KorGgenT) [reply data/json/items/ammo/8x40mm.json:15]: i suggest that the plural form is also 8x40mm caseless

Comment (Qrox) [inline data/json/items/ammo/8x40mm.json:15]:
```suggestion
    "name": { "str_sp": "8x40mm caseless" },
```

Comment (Qrox) [reply data/json/items/ammo/8x40mm.json]: Done.

**Pull request merged**